libs/server/ACL/ACLParser.cs

```diff
@@ -97,28 +97,28 @@ public static void ApplyACLOpToUser(ref User user, string op)
                 return;
             }
 
-            if (op == "on")
+            if (op.Equals("on", StringComparison.Ordinal))
             {
                 // Enable user
                 user.IsEnabled = true;
             }
-            else if (op == "off")
+            else if (op.Equals("off", StringComparison.Ordinal))
```

Comment:
While we are at it and we are turning these comparisons into case insensitive comparisons, we can also make the constant string operands upper-case to make the style consistent the rest of the code.